articles/install-guide/index.md

```diff
@@ -1,53 +1,225 @@
+
+1. Pre-requisites
+
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter notebooks](https://jupyter.readthedocs.io/en/latest/install.html)
+
+1. Install the `iqsharp` package:
+
+    ```bash
+    conda install -c quantum-engineering iqsharp
+    ```
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Run the following command to start the notebook server:
+
+        ```bash
+        jupyter notebook
+        ```
+
+    - Browse to the URL shown on the command line. For example: http://localhost:8888
+
+    - Create a jupyter notebook with a Q# kernel, and add the following code to the first notebook cell:
+
+        ```qsharp
+        operation HelloQ () : Unit {
+            Message($"Hello from quantum world!");
+        }
+        ```
+
+    - Run this cell of the notebook:
+
+        You should see `HelloQ` in the output of the cell. When running in jupyter notebooks, the Q# code is compiled, and the notebook outputs the name of the operation(s) that it finds.
+
+## Develop with C# on Windows, using Visual Studio
+
+1. Pre-requisites
+
+    - Visual Studio, version, with the .NET Core cross-platform development workload enabled
+
+1. Install the Q# Visual Studio extension
+
+    - Download the [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+    - Add the extension to Visual Studio
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Create a new Q# application
+
+        - Go to **File** -> **New** -> **Project**
+        - Type `Q#` in the search box
+        - Select **Q# Application**
+        - Select **Next**
+        - Choose a name and location for your application
+        - Select **Create**
+
+    - Inspect the project
+
+        You should see that two files have been created: `Driver.cs` which is the C# host application; and `Operation.qs`, which is a Q# program that defines a simple operation to print a message to the console.
+
+    - Run the application
+
+        - Select **Debug** -> **Start Without Debugging**
+        - You should see the text `Hello quantum world!` printed to a console window.
+
+## Develop with C#, using VS Code
+
+1. Pre-requisites
+
+   - [VS Code](https://code.visualstudio.com/download)
+   - [.NET Core SDK 2.1 or later](https://www.microsoft.com/net/download)
+
+1. Install the Quantum VS Code extension
+
+    - Download the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode)
+    - Add the extension to VS Code
+
+1. Install the Quantum project templates:
+
+    ```bash
+    dotnet new -i Microsoft.Quantum.ProjectTemplates
+    ```
+
+    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
+
+1. Verify the installation by creating a `Hello World` application
+
+    - Create a new project:
+
+        - Go to **View** -> **Command Palette**
```

Comment:
If I recall, the VS Code tutorial teaches the command palette can be accessed with Ctrl+Shift+P or ⌘+Shift+P, would that be reasonable to mention here?

Reply:
I guess I was going for the most agnostic way to present the information, to reduce clutter. When you execute this command, you can see the shortcut for your particular platform. And if you already know it, you already know it. Does that make sense?

Reply:
Definitely, that makes a lot of sense!